### PR TITLE
Some basic Korean text updates

### PR DESCRIPTION
**Pull request by bpaduch**
Re-using existing wordings + a little help from Google Translate. :)

@techhtml: Can you confirm my updates are correct?

### CODE REVIEW

File: translations/ko/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -23,7 +23,7 @@ msgstr "성공으로 가는 길"
 
 #: /content/learn/who/amp-ads.yaml
 msgid "AMP Ads"
-msgstr ""
+msgstr "AMP 광고는"
```
Comment (techhtml) [inline translations/ko/LC_MESSAGES/messages.po]: AMP 광고

File: translations/ko/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -983,23 +983,23 @@ msgstr ""
 
 #: /content/includes/menu.yaml
 msgid "Videos"
-msgstr ""
+msgstr "비디오"
 
 #: /content/docs/build.md
 msgid "View AMP Components"
-msgstr ""
+msgstr "보기 AMP 컴포넌트"
```
Comment (techhtml) [inline translations/ko/LC_MESSAGES/messages.po]: AMP 컴포넌트 보기

File: translations/ko/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -983,23 +983,23 @@ msgstr ""
...
 
 #: /content/docs/build.md
 msgid "View Guides"
-msgstr ""
+msgstr "보기 가이드"
```
Comment (techhtml) [inline translations/ko/LC_MESSAGES/messages.po]: 가이드 보기

File: translations/ko/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -983,23 +983,23 @@ msgstr ""
...
 
 #: /content/docs/build.md
 msgid "View Tutorials"
-msgstr ""
+msgstr "보기 튜토리얼"
```
Comment (techhtml) [inline translations/ko/LC_MESSAGES/messages.po]: 튜토리얼 보기

File: translations/ko/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -983,23 +983,23 @@ msgstr ""
...
 
 #: /content/docs/build.md
 msgid "View examples"
-msgstr ""
+msgstr "View 예제"
```
Comment (techhtml) [inline translations/ko/LC_MESSAGES/messages.po]: 예제 보기

**Comment by bpaduch** [comment]
@techhtml  - Thanks.  I've fixed them.  All good?

**Comment by techhtml** [comment]
LGTM :-)
Nice work!!!

**Comment by bpaduch** [comment]
@techhtml - thanks.  Can you mark the review as "approved".

**Pull request merged**